internal/api/ui/login/static/templates/main.html

```diff
@@ -8,13 +8,24 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
 
     <link rel="stylesheet" href="{{ resourceThemeUrl "css/zitadel.css" .Theme }}" type="text/css">
-    <link rel="icon" type="image/x-icon" href="{{ resourceThemeUrl "favicon.ico" .Theme }}">
+    
     {{ if hasCustomPolicy .LabelPolicy }}
     <link rel="stylesheet" href="{{ variablesCssFileUrl .PrivateLabelingOrgID .LabelPolicy}}" type="text/css">
     {{ end}}
+
+    {{ if hasCustomPolicy .LabelPolicy }}
```

Comment:
there's the exact same check just above... i'd merge them

Reply:
done